lib/auth/notification_test.go

```diff
@@ -229,7 +230,7 @@ func TestNotifications(t *testing.T) {
 	for _, n := range testNotifications {
 		// We add a small delay to ensure that the timestamps in the generated UUID's are all different and in the correct order.
 		// This is to prevent flakiness caused by the notifications being created in such quick succession that the timestamps are the same.
-		fakeClock.Advance(50 * time.Millisecond)
+		time.Sleep(50 * time.Millisecond)
```

Comment:
`fakeClock.Advance` wasn't working as intended because the `uuid` package doesn't use the clock to generate the timestamp.